liblustreapi-sys/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+include!("bindings.rs");
+
+type lstat_t = libc::stat64;
+pub const IOC_MDC_GETFILEINFO: u32 = 0xc0086916;
```

Comment:
How likely is this to change? Also, why not get this from bindgen?

Reply:
lstat_t defined to libc::stat64 is more correct and helps clean up a lot of the redefines from bindgen.
IOC_MDC_GETFILEINFO shouldn't change. It would be a Lustre regression if it does (c.f. [LU-8330](https://jira.whamcloud.com/browse/LU-8330)).  I could not get bindgen to cleanly generate the pub const, so this seemed cleaner.